src/templates/next/components/Divider/Divider.tsx

```diff
@@ -0,0 +1,7 @@
+import type { DividerProps } from "~/common"
+
+const Divider = ({}: DividerProps) => {
+  return <hr className="bg-divider-medium my-6" />
```

Comment:
do we need the `w-full` or similar width control to take the full width of the parent container?

Reply:
Hmm, the `<hr />` tag [automatically has](https://stackoverflow.com/a/21921785) a `width: 100%` set, so we don't need to specify further here.